### PR TITLE
Make referenced_by gives total count & includes docs

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: web/files/references.go [hunk 1]
```diff
@@ -28,10 +50,21 @@ func listReferencesHandler(c echo.Context) error {
...
+
+	var resCount couchdb.ViewResponse
+	err = couchdb.ExecView(instance, consts.FilesReferencedByView, reqCount, &resCount)
+	if err != nil {
+		return err
 	}
 
+	count := 0
+	if len(resCount.Rows) > 0 {
+		count = int(resCount.Rows[0].Value.(float64))
```
Comment (nono) [inline web/files/references.go]: Is the `.(float64)` necessary?

Comment (aenario) [inline web/files/references.go]: Yes, Row.Value is now an interface{} (we had to change it from your initial view implementation to support other view results).

**Comment by nono** [comment]
@aenario can you rebase master to check that travis build is really green?

**Comment by aenario** [comment]
done

**Pull request merged**